### PR TITLE
New Use Case: Zeit Now Continuous Deployment

### PR DESCRIPTION
**Pull request by TomFern**
New use case doc for demo project: https://github.com/semaphoreci-demos/semaphore-demo-zeit-now

### CODE REVIEW

File: use-case-zeit-now-ci-cd.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    pipeline_file: deploy-production.yml
+    auto_promote_on:
+      - result: passed
+        branch:
+          - master
+```
+
+Just 2 blocks make the CI pipeline:
+
+![CI Pipeline](https://github.com/semaphoreci-demos/semaphore-demo-zeit-now/raw/master/images/semaphore-zeit-now-ci.png)
```
Comment (markoa) [inline use-case-zeit-now-ci-cd.md]: Let's have one image of the entire workflow, with expanded deployment to production. That image should, like on other pages, appear at the end of the _Overview of the CI/CD pipeline_ section.

File: use-case-zeit-now-ci-cd.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+Both
+[nvm](https://docs.semaphoreci.com/article/32-ubuntu-1804-image#javascript-via-node-js)
+and [npm](https://www.npmjs.com) are pre-installed on Semaphore's Ubuntu
+image. We can use the them to switch Node.js versions and install
+packages.
+
+### Deployment
+
+[Zeit Now](https://zeit.co/) is a cloud service for serverless websites
+and web applications. Deployment is done with their [command
```
Comment (markoa) [inline use-case-zeit-now-ci-cd.md]: Let's refrain from saying "our" or "their" anywhere in our content. Simply say "Deployment is performed with the Now CLI ..."

**Comment by TomFern** [comment]
Thanks for the feedback! I've made the corrections.

**Comment by markoa** [comment]
Thanks!

Btw removing the intro sentence wasn't necessary, I'll add it back during publishing.

**Pull request merged**